common/scripted_triggers/wc_government_triggers.txt

```diff
@@ -0,0 +1,7 @@
+uses_clan_government_trigger = {
+    OR = {
+		culture_group = culture_group:orc_group
+        culture_group = culture_group:dwarf_group
+        culture_group = culture_group:tauren_group
+	}
```

Comment:
Also, mogu, centaur should probably have clans.
Also, `wastewander` human culture.